lib/_sidebar.erb

```diff
@@ -13,14 +13,28 @@
                 <%- else -%>
                   <%- active_keyword="" -%>
                 <%- end -%>
-                <li class="<%=active_keyword%>"><a href="<%=key2["slug"]%>.html" data-toggle="tab"><%=key2["title"]%></a></li>
+                <li class="<%=active_keyword%>"><a href="#<%=key2["slug"]%>" data-toggle="tab"><%=key2["title"]%></a></li>
               <%- end -%>
             <%- end -%>
           </ul>
         </nav>
       </div>
       <!-- Tab content -->
       <div class="tab-content">
-        <div class="tab-pane active text-style" id="content"></div>
+      <%- @site_map['docs'].each do |key, value| -%>
+        <%- first_entry = true -%>
+        <%- value.each do |key2, value2| -%>
+          <%- if key===@site_map.keys[0] && first_entry -%>
+            <%- active_keyword="active tab-pane text-shtmltyle" -%>
+            <%- first_entry = false -%>
+          <%- else -%>
```

Comment:
1. You don't need the `else` part of the if clause. It is just 2 lines shorter this way.

Combining this comment together with the comment above about line# 26, the code becomes:
```
            <%- value.keys.each do |key2| -%>
                <%- active_keyword="tab-pane text-style" -%>
                <%- if key===@site_map.keys[0] && first_entry -%>
                    <%- active_keyword="active tab-pane text-shtmltyle" -%>
                    <%- first_entry = false -%>
                <% - end -%>
                <div role="tabpanel" class="<%=active_keyword%>" id="<%=key2["slug"]%>">
                    <%- linked_doc = key2["link"] -%>
                    <%=  reveal(linked_doc) -%>
                </div>
            <%- end -%>
```